.gitignore

```diff
@@ -199,4 +199,7 @@ launchSettings.json
 !test/EndToEnd/ProjectTemplates/**/*.pfx
 
 .idea
-.ionide
+.ionider
+
+# Image manifests are generated at build-time for VSIX
+*.imagemanifest
```

Comment:
rather than ignoring the file, can we get t4 to output the file to our project's obj/ folder?
https://stackoverflow.com/a/26182408

Reply:
For this release, is it ok if I don't? And create a followup issue for next iteration?